lib/autobot.ts

```diff
@@ -140,11 +133,11 @@ export class Autobot {
         .filter(({ plugin }) => plugin.scope === scope)
         .map(
           ({ name, plugin: Plugin }) =>
-            (this.plugins[name] = {
```

Comment:
I don't actually know why I was setting the plugins here... It's more than a bit redundant.